README.md

```diff
@@ -13,8 +24,9 @@ The following environment variables can be used to configure lemmy-ui:
 | `ENV_VAR`                      | type     | default          | description                                                                         |
 | ------------------------------ | -------- | ---------------- | ----------------------------------------------------------------------------------- |
 | `LEMMY_UI_HOST`                | `string` | `0.0.0.0:1234`   | The IP / port that the lemmy-ui isomorphic node server is hosted at.                |
-| `LEMMY_UI_LEMMY_INTERNAL_HOST` | `string` | `0.0.0.0:8536`   | The internal IP / port that lemmy is hosted at. Often `lemmy:8536` if using docker. |
-| `LEMMY_UI_LEMMY_EXTERNAL_HOST` | `string` | `0.0.0.0:8536`   | The external IP / port that lemmy is hosted at. Often `DOMAIN.TLD`.                 |
+| `LEMMY_UI_BACKEND_REMOTE`      | `string` |                  | Domain of a remote Lemmy instance to connect using HTTPS and port 403               |
```

Comment:
Is this one optional? Might be good to have `undefined` in the default column, and maybe move this below the others.

Also would be good to add a note that its only necessary if you're connecting to a remote server.